substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -1865,6 +1865,24 @@ pub mod pallet {
 		MinBalanceDeficitAdjusted { pool_id: PoolId, amount: BalanceOf<T> },
 		/// Claimed excess frozen ED of af the reward pool.
 		MinBalanceExcessAdjusted { pool_id: PoolId, amount: BalanceOf<T> },
+		/// A pool member's claim permission has been updated.
+		MemberClaimPermissionUpdated { member: T::AccountId, permission: ClaimPermission },
+		/// A pool's metadata was updated.
+		MetadataUpdated { pool_id: PoolId, caller: T::AccountId },
+		/// A pool's nominating account (or the pool's root account) has nominated a validator set on behalf of the
+		/// pool.
+		PoolNominationMade { pool_id: PoolId, caller: T::AccountId },
+		/// A pool's nominator was chilled.
+		PoolNominatorChilled { pool_id: PoolId, caller: T::AccountId },
+		/// Global parameters regulating nomination pools have been updated.
+		GlobalNomPoolParamsUpdated {
```

Comment:
You can remove the redundant `NomPool` from the name since "pallet" already implies it. 
```suggestion
		GlobalParamsUpdated {
```